src/components/CheckBox/CheckboxList.tsx

```
@@ -0,0 +1,312 @@
+ * </CheckboxList>
+ * ```
+ */
+const CheckboxList = forwardRef<HTMLDivElement, CheckboxListProps>(
+  (
+    {
+      values: propValues,
+      defaultValues = [],
+      onValuesChange,
+      name: propName,
+      disabled = false,
+      size = 'medium',
```

ESLint

[error] 'size' is assigned a value but never used. Allowed unused args must match /^_/u.

(@typescript-eslint/no-unused-vars)